public/js/login.js

```diff
@@ -0,0 +1,33 @@
+const inputTags = document.querySelectorAll('#login-form input')
+const submitBtn = document.querySelector('#login-submit-btn');
+
+// 입력란을 모두 채웠는지 확인
+function checkAllInputFill() {
```

Comment:
요 js 구현 부분은 다음 과제인걸로 알고 있는데 미리 해주셨군요? 엄청 빠르시네요 👏 

netlify 에서 동작 확인했을 때도 잘 작동하고, 기본적인 DOM 이벤트 핸들러 등록에 대한 이해는 충분하신 것 같은데요,
세부적인 로직에서 살짝 놓치신 부분이 있는 것 같아서 아래 이어서 코멘트 남겨보겠습니다.

Reply:
-  isAllFill 이란 boolean 값은 default 값이 true 입니다. 물론 지금은 input 태그에 required 설정이 있기 때문에 아무것도 입력하지 않으면 로그인 버튼이 disabled 되긴 하지만, 만약 유저가 개발자도구를 열어서 required 속성을 지워버리면 로그인 버튼이 눌리는 상황이 발생할 수 있을 것 같아요. html의 속성을 활용하는 것도 아주 좋은 접근방법이지만 js 로직도 놓치는 부분 없게 작성해야 의도하지 않은 버그 발생을 방지할 수 있을거에요.

Reply:
- 현재 로직 상으로는 유저가 스페이스로 빈 string 값을 입력할 시에 로그인 버튼이 활성화될 것 같네요. 이런 경우를 방지하기 위해 보통 input validation 에서는 String.trim() 메소드를 사용하곤합니다.

Reply:
- 저는 개인적으로 필요한 경우가 아니면 let 을 사용하지 않으려고 하는 편인데요, 변수를 수동으로 재할당 해주는게 코드가 길어지기 쉽고, 의도 파악이 다소 어려워질 수 있기 때문입니다. 저라면 이 부분 로직을 아래와 같이 조금 더 짧고 읽기 쉽게 작성할거 같아요. 
```suggestion
function checkAllInputFill() {
  const isAllFill = Array.from(inputTags).every(tag => tag.value.trim() !== '');

  submitBtn.classList.toggle('active', isAllFill);
}
```

every 메소드에 대해 한번 공부해보세요~ 자매품으로 some도 있습니다.

Reply:
함수명과 변수명은 아주 잘 작성해주셨네요 👍